lib/metadata/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 3.12.0 FATAL_ERROR)
+foreach(variant IN ITEMS server client)
+  add_library(
+    irods_lib_metadata_${variant}
+    OBJECT
+    "${CMAKE_CURRENT_SOURCE_DIR}/src/metadata.cpp"
+  )
+  target_link_libraries(
+    irods_lib_metadata_${variant}
+    PUBLIC
+    irods_lib_api_${variant}
+    irods_lib_core_${variant}
+    irods_filesystem_${variant}
+  )
+  target_include_directories(
+    irods_lib_metadata_${variant}
+    PUBLIC
+    "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
+    "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_BINARY_DIR}/lib/core/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/lib/core/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/lib/api/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/lib/hasher/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/lib/rbudp/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/server/core/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/server/api/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/server/icat/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/server/re/include>"
+    "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/server/drivers/include>"
```

Comment:
I think we can remove all these server lines. Try it and see if it still builds

Reply:
Indeed, it just built without help from any server includes.